camel-core/src/main/docs/eips/dead-letter-channel.adoc

```diff
@@ -384,7 +384,7 @@ The onPrepare is also available using the default error handler.
 *Available as of Camel 2.10.4/2.11*
 
 When Camel error handler handles an error such as
-link:dead-letter-channel.html[Dead Letter Channel] or using
+<<DeadLetterChannel-DeadLetterChannel,Dead Letter Channel>> or using
```

Comment:
not sure if we have an adoc file for those kind of references. are you planning to provide or we will keep them like this?

Reply:
It would be better to have them already in the form of a link and fix them each time we find something missing, than revisit all the possible link one shot. So I'd go with this direction.

Reply:
that's all right. documentation refinement will never end anyway :)

Reply:
Well said :-)

Reply:
I found this https://issues.apache.org/jira/browse/CAMEL-12246, where @fbolton is trying to use those cross references for linking adocs. That's why I followed this. We might need to discuss about what are the expected final outcomes for the documentation.

Reply:
The beauty of this style of anchor:
```
<<DeadLetterChannel-DeadLetterChannel,Dead Letter Channel>>
```
Which references the anchor:
```
[[DeadLetterChannel-DeadLetterChannel]]
```
In the file:
https://raw.githubusercontent.com/apache/camel/master/camel-core/src/main/docs/eips/dead-letter-channel.adoc

Is that the link does not break if you rename the target file or if you move the target file to a different directory in the repo. Using the anchor reference gives you a level of abstraction that makes the link far less brittle.